js/renovation/ui/__tests__/check_box.test.tsx

```diff
@@ -12,6 +13,7 @@ import {
 } from '../check_box';
 import { Widget } from '../common/widget';
 import { InkRipple } from '../common/ink_ripple';
+import { ValidationMessage } from '../validationMessage';
```

Comment:
I suggest to mock real validationMessage
```
jest.mock('../validationMessage', () => ({ ValidationMessage: jest.fn() }));
```

Reply:
sorry, but why?

Reply:
I mocked this way: `jest.mock('../validationMessage');`